src/content/docs/en/guides/fonts.mdx

```diff
@@ -9,22 +9,22 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
 
 
 
-Astro supports all common strategies for adding custom typefaces to your site design. This guide will show you two different options for including web fonts in your project.
+Astro supports all common strategies for adding custom typefaces to your site design. This guide will show you two options for including web fonts in your project.
 
-## Using a local font file
+## Add fonts to your project
+
+### Using a local font file
 
 If you want to add font files to your project, we recommend adding them to your [`public/` directory](/en/core-concepts/project-structure/#public). In your CSS you can then register fonts with an [`@font-face` statement](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face) and use the `font-family` property to style your site.
```

Comment:
```suggestion
```
I'm thinking we don't need this introduction at all - the example steps just repeat this content.